eng/pipelines/templates/variables/vmr-build.yml

```diff
@@ -17,6 +17,13 @@ variables:
   - name: VmrBranch
     value: ${{ replace(replace(variables['Build.SourceBranch'], 'refs/heads/', ''), 'refs/pull/', '') }}
 
+- ${{ if and(eq(variables['System.TeamProject'], 'internal'), ne(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
We now have a `isOfficialBuild` variable below. You might want to use that.

Reply:
Good call. Still verifying that the functionality works as expected at this point.